bitnami/ghost/templates/_helpers.tpl

```diff
@@ -120,10 +108,41 @@ Return the MariaDB Secret Name
 */}}
 {{- define "ghost.databaseSecretName" -}}
 {{- if .Values.mariadb.enabled }}
-    {{- printf "%s" (include "ghost.mariadb.fullname" .) -}}
+    {{- if .Values.mariadb.auth.existingSecret -}}
+        {{- printf "%s" .Values.mariadb.auth.existingSecret -}}
```

Comment:
This is not in values.yaml. Even though it is a subchart I would add it to the values.yaml and to the values table